api/net/Areas/Admin/Controllers/AlertController.cs

```diff
@@ -0,0 +1,109 @@
+using System.Net;
+using System.Net.Mime;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using TNO.API.Areas.Admin.Models.Alert;
+using TNO.API.Models;
+using TNO.DAL.Services;
+using TNO.Entities;
+using TNO.Keycloak;
+
+namespace TNO.API.Areas.Admin.Controllers;
+
+/// <summary>
+/// AlertController class, provides alert endpoints for the admin api.
+/// </summary>
+[ApiController]
+[Area("admin")]
+[ApiVersion("1.0")]
+[Route("api/v{version:apiVersion}/[area]/alerts")]
+[Route("api/[area]/alerts")]
+[Route("v{version:apiVersion}/[area]/alerts")]
+[Route("[area]/alerts")]
+[ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.Unauthorized)]
+[ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.Forbidden)]
+public class AlertController : ControllerBase
```

Comment:
Mark controller for admin role `[ClientRoleAuthorize(ClientRole.Administrator)]`.  Make a separate controller in the editor area.  Having an anonymous endpoint here is misleading.